lib/sidekiq/web/application.rb

```diff
@@ -291,6 +292,26 @@ def call(env)
       resp
     end
 
+    def get_csp_header(scheme)
```

Comment:
This method has two possible inputs, http and https, but nothing else dynamic so it only has two possible outputs. What do you think about memoizing the two constructed policies to minimize the runtime overhead?

What should it do in the case of a non-http scheme?  Is that even possible?

Reply:
memoizin: can do.

in case of non-http: it should probably not include the header. Will update for that case.